.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,18 @@
+---
+name: Feature/Change Request
+about: A template for Druid feature and change requests
+title: "[REQUEST]"
+labels: Wish list
+assignees: ''
+
+---
+
+# Description
+
+Please describe the feature or change with as much detail as possible.
```

Comment:
Maybe add something about how if you have a detailed implementation in mind, and you're wanting to contribute it yourself, "Proposal" is better than "Feature request".

Reply:
Added a note on proposals